README.md

```diff
@@ -95,18 +95,23 @@ pip install .
 To build Modulus docker image:
 
 ```bash
-docker build -t modulus:deploy --target deploy -f Dockerfile .
+docker build -t modulus:deploy \
+    --build-arg TARGETPLATFORM=linux/amd64 --target deploy -f Dockerfile .
 ```
 
 Alternatively, you can run `make container-deploy`
 
 To build CI image:
 
 ```bash
-docker build -t modulus:ci --target ci -f Dockerfile .
+docker build -t modulus:ci \
+    --build-arg TARGETPLATFORM=linux/amd64 --target ci -f Dockerfile .
 ```
 
-Alternatively, you can run `make container-ci`
+Alternatively, you can run `make container-ci`.
+
+Currently only `linux/amd64` and `linux/arm64` platforms are supported. `linux/amd64`
+is the recommended default.
```

Comment:
Maybe leave out the recommended default? It's not like there is a choice to make once you already have your system.

Reply:
Good point haha!

Reply:
Done! Also added a note about `warp-lang` support on arm64